### PR TITLE
New package: o2sh.onefetch version 2.12.0

### PR DESCRIPTION
**Pull request by o2sh**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/61982)

### CODE REVIEW

File: manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS0.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: o2sh.onefetch
+PackageVersion: 2.12.0
```
Comment (ItzLevvie) [inline manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.installer.yaml:5]:
```suggestion
PackageVersion: 1.5
```
This must match Control Panel.

File: manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS0.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: o2sh.onefetch
+PackageVersion: 2.12.0
```
Comment (ItzLevvie) [inline manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.5
```
This must match Control Panel.

File: manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS0.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: o2sh.onefetch
+PackageVersion: 2.12.0
```
Comment (ItzLevvie) [inline manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.yaml:5]:
```suggestion
PackageVersion: 1.5
```
This must match Control Panel.

File: manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+PackageLocale: en-US
+Publisher: Ossama Hjaji
```
Comment (ItzLevvie) [inline manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.locale.en-US.yaml:7]:
```suggestion
Publisher: o2sh
```
This must match Control Panel.

File: manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+PublisherUrl: https://github.com/o2sh/
+PublisherSupportUrl: https://github.com/o2sh/onefetch/issues
+# PrivacyUrl: 
+Author: Ossama Hjaji
+PackageName: onefetch
```
Comment (ItzLevvie) [inline manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.locale.en-US.yaml:12]:
```suggestion
PackageName: Onefetch
```
This must match Control Panel (i.e. capitalization of the `o`)

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.locale.en-US.yaml (hunk 2 above)
Comment (o2sh) [reply manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.locale.en-US.yaml:5]: I have updated the installer to match the manifest.

File: manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.locale.en-US.yaml (hunk 5 above)
Comment (o2sh) [reply manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.locale.en-US.yaml:12]: I have updated the installer to match the manifest.

File: manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.installer.yaml (hunk 1 above)
Comment (o2sh) [reply manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.installer.yaml:5]: I have updated the installer to match the manifest.

**Comment by o2sh** [comment]
Hi @ItzLevvie Thanks for you remarks.
I have updated the installer to match the manifest.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**